### DISCUSSION TITLE
Error: ReferenceError: ReadableStream is not defined

### DISCUSSION THREAD

**Discussion by hygormartins10** [nodejs/undici#2508] [Q&A]
After updating the version of the "[undici](https://github.com/nodejs/undici)" lib version to 6.0.1 we started receiving the error "ReferenceError: ReadableStream is not Define", has anyone experienced this?

**Comment by ronag** [comment]
Note that you need at least node 18

**Reply by hygormartins10** [reply]
We are using node version 18.17.0

**Reply by ronag** [reply]
Please make bug report in the issue tracker.
